alma_sbom.py

```diff
@@ -79,7 +88,7 @@ def __call__(self, sbom_type_file_format: str) -> FileFormat:
         )
 
     @classmethod
-    def choices(cls) -> list[FileFormat]:
+    def choices(cls) -> typing.List[FileFormat]:
```

Comment:
`List[FileFormat]` as was explicitly imported in line 7

Reply:
I just fixed this commit